crates/network-tunnel/src/sshforwarding.rs

```diff
@@ -80,6 +87,13 @@ pub struct SshForwarding {
     process: Option<Child>,
 }
 
+fn split_host_port(hostport: String) -> Option<(String, u16)> {
+    let mut splits = hostport.as_str().splitn(2, ':');
```

Comment:
Should this be `rsplitn`, since we might have username and password in the host? e.g.:

```
mahdi@host.com:8080
```

Reply:
Currently none of the connectors for which this logic will trigger support username/password in the `address` field. I'd be open to adding that, but I think it would require some additional code to chop off anything up to an `@` character and then reattach that to the rewritten localhost addres.